ast/edit/testcase_test.go

```diff
@@ -74,3 +81,134 @@ myVar = 4`
 		t.Errorf("unexpected package count: want: 0, got: %d", len(transformed))
 	}
 }
+
+func TestTestcaseTransformImport(t *testing.T) {
+	fs := &inmemFS{
+		files: map[string]string{
+			"a/a_test.flux": `package a_test
+import "testing/assert"
+want = 4
+testcase a {
+	assert.equal(want: want, got: 2 + 2)
+}
+`,
+			"b/b_test.flux": `package b_test
+import "testing/assert"
+testcase b {
+	import "../a/a_test"
+	a_test.a()
+	assert.equal(want: 6, got: 3 + 3)
+}
+`,
+		},
+	}
+
+	ctx := filesystem.Inject(context.Background(), fs)
+	data, err := filesystem.ReadFile(ctx, "b/b_test.flux")
+	if err != nil {
+		t.Fatal(err)
+	}
+	pkg := parser.ParseSource(string(data))
+	if ast.Check(pkg) > 0 {
+		t.Fatalf("unexpected error parsing b/b_test.flux: %s", ast.GetError(pkg))
+	}
+	pkg.Files[0].Name = "b/b_test.flux"
+
+	names, pkgs, err := edit.TestcaseTransform(ctx, pkg)
+	if err != nil {
+		t.Fatalf("unexpected error: %s", err)
+	}
+
+	if want, got := []string{"b"}, names; !cmp.Equal(want, got) {
+		t.Fatalf("unexpected testcase names -want/+got:\n%s", cmp.Diff(want, got))
+	}
+
+	files := make([]string, len(pkgs[0].Files))
+	for i, file := range pkgs[0].Files {
+		files[i] = ast.Format(file)
+	}
+
+	want := []string{
```

Comment:
Yesssss. Wondered if this test was going to assert on AST data structure or source and glad to see source as the "want." Makes the intent much more clear.

Reply:
Yea, I get really tired of trying to type out AST data structures so I've gravitated to using the formatter more often. Pretty sure it's going to bite me in the rear whenever we need/want to make changes to the formatter output, but we'll just have to take that since this is a lot easier to read.